internal/constants/scripts.go

```diff
@@ -56,6 +56,10 @@ else
   ETC_NETWORK_MANAGER="/etc/NetworkManager/system-connections"
 fi
 
+# remove default connection file create by NM(nm-initrd-generator). This is a WA until
```

Comment:
Maybe reference the bug you're adding this work-around for, so that later it's clear what to remove.